src/pages-dynamic/colorway.js

```diff
@@ -26,10 +26,30 @@ const Maker = (props) => {
     setState({ text: 'Copied!' });
   };
 
-  const [wishlist, setStateWishlist] = useState(undefined);
+  const [wishlistContainer, setStateWishlist] = useState({
```

Comment:
why not setting the state to undefined? Because we need to access `container.wishlists` ?